lib/app/screens/home.dart

```diff
@@ -70,10 +72,10 @@ class FlightPlanList extends ConsumerWidget {
             children: [
               GSButton(
                 onPressed: () {
-                  warningToast(
-                    context: context,
-                    title: 'Unimplemented',
-                    message: "We'll get around to this soon!",
+                  Navigator.push(
+                    context,
+                    MaterialPageRoute(
+                        builder: (context) => const AddTripPage()),
                   );
```

Comment:
Add this to the router, otherwise routing is gonna break. This is legacy style code.